site/specification/ORCv1.md

```diff
@@ -886,6 +886,12 @@ uses three streams PRESENT, DATA, and LENGTH, which stores the length
 of each value. The details of each type will be presented in the
 following subsections.
 
+Note that the order of these streams is **not fixed**. In the example
+of the integer column mentioned above, the order of the PRESENT stream
+and the DATA stream cannot be determined in advance. Instead, we need
+to determine the type of stream based on the Stream Kind, rather than
```

Comment:
Should we mention the following:
- index and data streams cannot be interleaved.
- the order of streams from different columns is not fixed as well.